packages/pipeline/test/parsers/copper/index_test.ts

```diff
@@ -0,0 +1,84 @@
+import * as chai from 'chai';
+import 'mocha';
+
+import {
+    CopperActivity,
+    CopperActivityType,
+    CopperCustomField,
+    CopperLead,
+    CopperOpportunity,
+} from '../../../src/entities';
+import {
+    CopperActivityResponse,
+    CopperActivityTypeCategory,
+    CopperActivityTypeResponse,
+    CopperCustomFieldResponse,
+    CopperSearchResponse,
+    parseActivities,
+    parseActivityTypes,
+    parseCustomFields,
+    parseLeads,
+    parseOpportunities,
+} from '../../../src/parsers/copper';
+import { chaiSetup } from '../../utils/chai_setup';
+
+chaiSetup.configure();
+const expect = chai.expect;
+
+type CopperResponse = CopperSearchResponse | CopperCustomFieldResponse;
+type CopperEntity = CopperLead | CopperActivity | CopperOpportunity | CopperActivityType | CopperCustomField;
+
+import * as activityTypesApiResponse from '../../fixtures/copper/api_v1_activity_types.json';
+import * as activityTypesParsed from '../../fixtures/copper/api_v1_activity_types.parsed.json';
+import * as customFieldsApiResponse from '../../fixtures/copper/api_v1_custom_field_definitions.json';
+import * as customFieldsParsed from '../../fixtures/copper/api_v1_custom_field_definitions.parsed.json';
+import * as listActivitiesApiResponse from '../../fixtures/copper/api_v1_list_activities.json';
+import * as listActivitiesParsed from '../../fixtures/copper/api_v1_list_activities.parsed.json';
+import * as listLeadsApiResponse from '../../fixtures/copper/api_v1_list_leads.json';
+import * as listLeadsParsed from '../../fixtures/copper/api_v1_list_leads.parsed.json';
+import * as listOpportunitiesApiResponse from '../../fixtures/copper/api_v1_list_opportunities.json';
+import * as listOpportunitiesParsed from '../../fixtures/copper/api_v1_list_opportunities.parsed.json';
+
+interface TestCase {
```

Comment:
I really like how these tests are structured. Makes it very easy to read and update later on 😄